index.js

```diff
@@ -2,17 +2,44 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+//made js render after body loads
 
 /***** Deliverable 2 *****/
-
+header.style.color="#abcdef"
 
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
+// travelerNameNode=document.querySelector("h2")
+// travelerNameNode.innerHTML=traveler.name
+
+// travelerNicknameNode=document.querySelector("div em")
+// console.log(travelerNameNode)
+//travelerNameNode.innerHTML=traveler.nickname
+//^ doesn't work, em is inline
+
+profileDiv=document.querySelector("#profile")
+profileDiv.innerHTML=`<img src="${traveler.photo}"
+alt="${traveler.name}">
+<h2>${traveler.name}</h2>
+<em>${traveler.nickname}</em>`
 
 
 /***** Deliverable 4 *****/
+var sightings=traveler.animalSightings
```

Comment:
Stick with `let` or `const`. Throw `var` away! 🚮

```suggestion
const sightings=traveler.animalSightings
```